src/protocol/models.ts

```diff
@@ -34,12 +36,14 @@ export interface AuthorizationRequestBody {
   reason: string;
   callbackUrl: string;
   scope: ZKPRequest[];
+  did_doc?: DIDDocument;
```

Comment:
Maybe better to rename DocumentDid?

Reply:
no, it's common term

Reply:
https://www.w3.org/TR/did-core/#dfn-did-documents